lib/components/util/externalLink.tsx

```diff
@@ -0,0 +1,63 @@
+import { ExternalLinkAlt } from '@styled-icons/fa-solid'
+import { useIntl } from 'react-intl'
+import React, { HTMLAttributes } from 'react'
+
+interface LinkProps extends HTMLAttributes<HTMLElement> {
+  contents: JSX.Element | string
+  gap?: number
+  isInline?: boolean
+  size?: number
+  style?: React.CSSProperties
+  url: string
+}
+
+interface IconProps extends HTMLAttributes<HTMLElement> {
+  size?: number
+  style?: React.CSSProperties
+}
+
+export const NewWindowIconA11y = ({
+  size = 14,
+  style
+}: IconProps): JSX.Element => {
+  const intl = useIntl()
+  return (
+    <ExternalLinkAlt
+      /* the "title" prop removes aria-hidden from styled-icons, so use the title as the aria-label as well
+        https://github.com/styled-icons/styled-icons#accessibility */
+      aria-labelledby="title"
+      height={size}
+      style={style}
+      title={intl.formatMessage({ id: 'common.linkOpensNewWindow' })}
+    />
+  )
+}
+
+export const LinkOpensNewWindow = ({
+  contents,
+  gap = 6,
+  isInline,
+  size = 14,
+  style,
+  url
+}: LinkProps): JSX.Element => {
+  return (
+    <a
+      href={url}
+      rel="noreferrer"
+      style={{
+        alignItems: 'center',
+        display: `${isInline ? 'inline-flex' : 'flex'}`,
+        flexDirection: 'row',
+        gap: `${gap}px`,
+        textDecoration: 'underline',
```

Comment:
Is this intentional? It's always underlined and there's no hover effect. This is really confusing and doesn't behave the way I'd expect a link to behave. Is there any way to make it only underline on hover?